src/User.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace J4k\OAuth2\Client\Provider;
+
+use League\OAuth2\Client\Provider\ResourceOwnerInterface;
+
+/**
+ * @see     https://vk.com/dev/fields
+ *
+ * @package J4k\OAuth2\Client\Provider
+ */
+class User implements ResourceOwnerInterface
+{
+    // ========== Interface ==========
+
+    /**
+     * @type array
+     */
+    protected $response;
+
+    /**
+     * User constructor.
+     *
+     * @param array $response
+     */
+    public function __construct(array $response)
+    {
+        $this->response = $response;
+    }
+    /**
+     * @return array
+     */
+    public function toArray()
+    {
+        return $this->response;
+    }
+    /**
+     * @return integer
+     */
+    public function getId()
+    {
+        return (int)($this->getField('uid') ?: $this->getField('id'));
+    }
+
+    // ========== helpers ==========
+
+    /**
+     * Helper for getting user data
+     *
+     * @param string $key
+     *
+     * @return mixed|null
+     */
+    protected function getField($key)
+    {
+        return !empty($this->response[$key]) ? $this->response[$key] : null;
+    }
+
+    // ========== Getters for default scope ==========
```

Comment:
@j4k, do you mean "not-dockblocks"? Sure, i think same. They're just meta-blocks that can be moved to trait-helpers, value-object-class, etc.